addon/components/rdfa/rdfa-editor.hbs

```diff
@@ -49,17 +49,44 @@
 
     <div class='say-container__aside'>
       <div class='say-editor-hints'>
-        <ul class='say-editor-hints__list'>
-          {{#each this.sidebarWidgets as |widget|}}
-            <li class='say-editor-hints__list-item'>
-              <div class='say-card'>
-                {{! Todo: remove this div once it is over }}
+        {{#if this.insertSidebarWidgets.length}}
+          <Rdfa::PluginWrapper @title="Invoegen">
+            <AuList @divider={{true}}>
+              {{#each this.insertSidebarWidgets as |widget|}}
                 {{component
                   widget.componentName
                   controller=widget.controller
                   plugin=widget.plugin
                 }}
-              </div>
+              {{/each}}
+            </AuList>
+          </Rdfa::PluginWrapper>
+        {{/if}}
+        <ul class='say-editor-hints__list'>
+          
+          {{#if this.suggestedHints.length}}
+            {{#each this.suggestedHints as |hint|}}
+              <li class='say-editor-hints__list-item'>
+                {{component hint.component info=hint.info closeHints=this.closeHints}}
+              </li>
+            {{/each}}
+          {{/if}}
+
+          {{#if this.hasActiveHints}}
```

Comment:
this block should be dropped (I think it's still there because of a faulty merge)